content/docs/rfcs/23/README.md

```diff
@@ -0,0 +1,155 @@
+---
+slug: 23
+title: 23/WAKU2-TOPICS
+name: Waku v2 Topic Usage Recommendations
+status: draft
+editor: Oskar Thoren <oskar@status.im>
+contributors:
+---
+
+This document outlines recommended usage of topics in Waku v2. In [10/WAKU2 spec](/spec/10) there are two types of topics:
+
+- Pubsub topics, used for routing
+- Content topics, used for content-based filtering
+
+## Pubsub topics
+
+Pubsub topics are used for routing of mesages, see [11/WAKU2-RELAY](/spec/11) spec for more details of how this routing works.
+
+As written in [10/WAKU2 spec](/spec/10) there is a default Pubsub topic:
+
+`/waku/2/default-waku/proto`
+
+This indicates that
+
+1) It relates to the Waku problem domain
+2) version is 2
+3) name indicates what is being exchanged, which in this case is WakuMessages over a single topic and
+4) that the data in PubSub field is protobuf (unlike Eth2 where it is `ssz_snappy`) as determined by WakuMessage.
+
+The default PubSub topic SHOULD be used for all protocols.
```

Comment:
I would probably move this section closer to

### Why use a single PubSub topic by default?

(or the other way around, or a link)

As I found it a bit confusing that first we give options (proto/szz_snappy), and then immediately tell them not to use them without an explanation, so I would probably get the two a bit closer so that it's easier to understand why you would want to stick to the default topic.

Reply:
Good point, moved section up earlier